openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST2/tables/config/MCConfigurationTable_def.txt

```diff
@@ -11,15 +11,8 @@ TABLE "MC_Configuration" BIG_ENDIAN KEY_VALUE "Memory Control Configuration Tabl
   APPEND_PARAMETER "Memory_Scrubbing" 8 UINT 0 1 1
     STATE DISABLE 0
     STATE ENABLE 1
+  # NOTE: Python tables do not support GENERIC_WRITE_CONVERSION
```

Comment:
The reason is that the table conversions take place in Ruby (we dispatch to the Ruby backend) but that makes them unusable in the Python APIs. A custom conversion class would have the same problem.